### PR TITLE
first version of model, controller,and template for registration

### PR DESCRIPTION
**Pull request by koushikkm**
This is the initial version of the account app

### CODE REVIEW

**Comment by koushikkm** [comment]
@pankajssp

File: account/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='BasicUser',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```
Comment (koushikkm) [inline account/migrations/0001_initial.py]: avoid id, we don't need it here.

File: account/migrations/0001_initial.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                ('password', models.CharField(max_length=128, verbose_name='password')),
+                ('last_login', models.DateTimeField(blank=True, null=True, verbose_name='last login')),
+                ('email', models.EmailField(max_length=254, unique=True, verbose_name='user email')),
```
Comment (koushikkm) [inline account/migrations/0001_initial.py]: make email primary id

File: account/forms.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from django.forms import ModelForm
+from .models import BasicUser
+
+#Note:this class has tight coupling with classes in models
+class BasicUserForm(ModelForm):
+    class Meta:
+        model = BasicUser
+        #Avoid password since we are generating it
+        fields = ['email', 'full_name', 'mobile_number']
```
Comment (pankajssp) [inline account/forms.py]: Pls check if email will come before full_name?

File: account/models.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+    #We want to generate the password ourselves
+    #TODO: need to figure out how we suppress password in the user form
+    #passwd = models.CharField("user password", max_length=200)
+
+    #Full name of the user
+    full_name = models.CharField("full name", max_length=500)
+   
+    #Mobile number of the user
+    mobile_regex = RegexValidator(regex=r'^\+?1?\d{9,15}$', message="Mobile number be entered in the format:'+999999999'.Up to 15 digits")
```
Comment (pankajssp) [inline account/models.py]: Should this validation go in the front-end part?

Comment (koushikkm) [inline account/models.py]: We can certainly  have validation in front end. We can later remove from backend, if it turns out to be redundant.

File: account/forms.py (hunk 3 above)
Comment (koushikkm) [inline account/forms.py]: This will be corrected.

**Comment by koushikkm** [comment]
@pankajssp : I have taken care of most of the review comments

**Pull request merged**